packages/mdc-select/_variables.scss

```diff
@@ -49,9 +48,11 @@ $mdc-select-disabled-ink-color: rgba(mdc-theme-prop-value(on-surface), .37) !def
 $mdc-select-disabled-fill-color: mix(mdc-theme-prop-value(on-surface), mdc-theme-prop-value(surface), 2%) !default;
 $mdc-select-disabled-dropdown-opacity: .37 !default;
 
+$mdc-select-outlined-idle-border: rgba(mdc-theme-prop-value(on-surface), .38) !default;
```

Comment:
Are these design approved? Sometimes the spec is not up to date.

Reply:
Yes, just talked with design about them.